src/WebJobs.Script/Description/Workers/WorkerFunctionDescriptorProvider.cs

```diff
@@ -90,5 +104,43 @@ protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath,
 
             return parameters;
         }
+
+        private bool BindingAttributeContainsExpression(IEnumerable<FunctionBinding> bindings)
+        {
+            BindingFlags flags = BindingFlags.Instance | BindingFlags.Public;
+
+            if (bindings.Count() <= 0)
+            {
+                return false;
+            }
+
+            foreach (ExtensionBinding binding in bindings)
+            {
+                if (binding.Metadata.IsTrigger)
+                {
+                    // skip triggers, we only care if input and output bindings contain expressions
+                    continue;
+                }
+
+                foreach (var attribute in binding.Attributes)
```

Comment:
Got sidetracked and forgot to send this. But we should be able to remove the `foreach` below and replace the where/select/toarray calls with one `Any` with the predicate and Regex evaluation. That should reduce a lot of the work being done here.